axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,194 @@ def strategy(self, opponent: Player) -> Action:
         else:
             # Play TFT
             return opponent.history[-1]
+
+class Tranquilizer(Player):
+   
+    """
+    Submitted to Axelrod's second tournament by Craig Feathers
+
+    This strategy is based on the reverse engineering of the 
+    Fortran strategy K67R from Axelrod's second tournament.
+    Reversed engineered by: Owen Campbell, Will Guo and Mansour Hakem.
+
+
+    Description given in Axelrod's "More Effective Choice in the 
+    Prisoner's Dilemma" paper: The rule normally cooperates but 
+    is ready to defect if the other player defects too often. 
+    Thus the rule tends to cooperate for the first dozen or two moves
+    if the other player is cooperating, but then it throws in a 
+    defection. If the other player continues to cooperate, then defections 
+    become more frequent. But as long as Tranquilizer is maintaining an 
+    average payoff of at least 2.25 points per move, it will never defect 
+    twice in succession and it will not defect more than 
+    one-quarter of the time.
+
+    
+    - Has a variable, 'FD' which can be 0, 1 or 2. It has an initial value of 0
```

Comment:
Can you remove all of this part of the docstring and replace it with something along the lines of:

```
The strategy starts by cooperating this strategy has 3 states.

At the start of the strategy it updates it's states:

- It counts the number of consecutive defections by the opponent.
- If it was in state 2 it moves to state 0 and calculates the following quantities [INCLUDE THE CALCULATIONS]
- If it was in state 1 it moves to state 2 and calculates the following quantities [INCLUDE THE CALCULATIONS]

If after this it is in state 1 or 2 then it cooperates.

If it is in state 0 it will potentially perform 1 of the 2 following stochastic tests:

1. If [CONDITION ON THE SCORE] then it calculates a value [DETAILS ABOUT CALCULATION OF probability] and will cooperate if a random sampled number is less than that value. If it does not cooperate then the strategy moves to state 1 and defects.
2. If [CONDITION ON THE SCORE] then it calculates a value  [DETAILS ABOUT CALCULATION OF probability] and will cooperate if a random sampled number is less than that value. If not, it defects.

If none of the above holds the player simply plays tit for tat.
```